### PR TITLE
Create initial Dutch translation

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

**Comment by IntelOrca** [comment]
first

**Comment by Gymnasiast** [comment]
I'm getting very lively flashbacks of 2014, when I started contributing to OpenRCT2, with exactly the same thing. :D

File: data/language/nl-NL.txt [hunk 1]
```diff
@@ -0,0 +1,2113 @@
...
+STR_0222    :Kan kleurenschema niet veranderen...
+STR_0223    :Inzoomen
+STR_0224    :Uitzoomen
+STR_0225    :Steden
+STR_0226    :Industrieën
+STR_0227    :Vliegveld
+STR_0228    :Haven
+STR_0229    :Opties voor landschapsgenerator
+STR_0230    :
+STR_0231    :±
```
Comment (AaronVanGeffen) [inline data/language/nl-NL.txt]: Is this correctly encoded?

Comment (Gymnasiast) [reply data/language/nl-NL.txt]: It's the same in en-US. Most likely just a special code that IntelOrca has not converted, like {STRINGID}, {EURO}, etc.

Comment (Gymnasiast) [reply data/language/nl-NL.txt]: ± has code 177 in CP1252, by the way.

**Comment by Gymnasiast** [comment]
@IntelOrca Would you merge this?

**Pull request merged**